srcpkgs/deepin-voice-recorder/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'deepin-voice-recorder'
+pkgname=deepin-voice-recorder
+version=1.3.7
+revision=1
+build_style=qmake
+hostmakedepends="qt5-qmake pkg-config"
+makedepends="qt5-multimedia-devel dtkwidget-devel ffmpeg-devel pulseaudio-devel"
+depends="qt5integration gst-plugins-good1"
+short_desc="Deepin voice recorder"
+maintainer="John <johnz@posteo.net>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/linuxdeepin/deepin-voice-recorder/"
+changelog="https://github.com/linuxdeepin/deepin-voice-recorder/blob/${version}/CHANGELOG.md"
```

Comment:
would be preferable to have the raw changelog so that it can be show in a terminal with curl.

Reply:
I would prefer to have Markdown files not as raw links since most users are probably using a  proper web browser to view them, and markdown is way more comfortable to read when shown properly.